notebooks/responsibleaidashboard/responsibleaidashboard-census-classification-model-debugging.ipynb

```diff
@@ -118,9 +118,13 @@
     "        ('num_pipe', num_pipe, pipe_cfg['num_cols']),\n",
     "        ('cat_pipe', cat_pipe, pipe_cfg['cat_cols'])\n",
     "    ])\n",
-    "    X = feat_pipe.fit_transform(X)\n",
-    "    print(pipe_cfg['cat_cols'])\n",
-    "    return X, feat_pipe, features, classes\n",
+    "\n",
+    "    # Append classifier to preprocessing pipeline.\n",
+    "    # Now we have a full prediction pipeline.\n",
+    "    pipeline = Pipeline(steps=[('preprocessor', feat_pipe),\n",
+    "                               ('model', LGBMClassifier(n_estimators=5))])\n",
```

Comment:
if you try to increase number of estimators, does it still make some errors?  If so, I would just recommend keeping it at default (100), otherwise I've seen some strange things before because there are so few trees, like a lot of variables don't make any different in what if analysis.

Reply:
Done